### ISSUE TITLE
Seem to be unable to rebind keys

### ISSUE THREAD

**Issue by 4444Stricture** [loopuleasa/Dota2-TheCore-Config-Engine#146]
Hi

I installed your configuration a couple of days ago and I must say I like it a lot, it is very intuitive. 
Kudos to you all, loopuleasa and team.

Basically I am a complete beginner when it comes to editing Dota 2's key-binds.

I am using the 'Loopuleasa's Super Compact QWERTG-DFXCV layout (ALT,SPACE mods)' version. Everything seems to work properly, beyond the expected problems due to me using a Swedish QWERTY keyboard, so I naturally ran into the whole ping issue, as well as the fact that I cannot open the console.

My issue is that I cannot edit any key-binds.
 
For example I try to remove the bind of the F1 key by erasing the line:
"
bind "F1" "+rune"
"
In the dota2_keybinds_default.cfg file, but pressing the F1 key still executes the rune script.
I have tried rebinding several other keys without result.

Which step am I missing?

**Comment by dodekeract** [comment]
DotA caches the last bindings you had if I remember correctly. You may be able to fix this by typing `unbind F1` in your DotA console.

**Comment by 4444Stricture** [comment]
Tried it and forced a reboot of autoexec.cfg, did not work.

Besides 'dota2_keybinds_default.cfg' contains the unbindall command. Does this work differently than unbind?

**Comment by dodekeract** [comment]
Not sure, but the autoexec parser/interpreter is quite weird and buggy, I wouldn't be surprised if they differ.

You should try to find the old bind in the non-autoexec files in your config folder. I think the cache is somewhere in these. Also you may want to try to backup your config folder (in case it doesn't work), then delete the config folder's contents and start DotA again.

No guarantee on that, but that's what I'd try.

**Comment by 4444Stricture** [comment]
I deleted the content of the folder SteamApps\common\dota 2 beta\dota\config
Was this the right folder?

F1 still executes rune script.

**Comment by dodekeract** [comment]
It most likely wasn't the right folder. Here is [the overview](https://manta.dodekeract.report) I created to explain where to find these files for Manta. You should find the real folder easily using it.

**Comment by 4444Stricture** [comment]
Right so I found out the issue, as embarrassingly simple as I though.

For some reason Dota or Steam created a duplicated folder of all of my game files in 'dota 2 beta\dota' as opposed to 'dota 2 beta\game\dota'. Fool that I was I tried to edit the duplicated files.

Did a clean installation and the duplicated folder is gone. The issues are gone as well.

Thank you for your attention dodekeract, sorry it was such a stupid thing :(
